### DISCUSSION TITLE
wifi card 4530s OSX Catalina

### DISCUSSION THREAD

**Discussion by MANALICANTE** [chris1111/HP-Probook-EliteBook-Package-Creator-OC#163] [General]
Hi Chris and other friends! i have installed OSX Catalina on my old HP 4530s, initially everything works, except sometimes it does something weird. But wifi does not work, this was already known. Can you tell me which whitelisted compatible wifi card for HP 4530s I can buy? Thank you very much for your incredible help.

Greetings from Spain!!

**Comment by chris1111** [comment]
For Catalina and the HP 4530s you can use the 

## Recommand WIFI
- Wifi mini PCI Wifi mini PCI [AzureWave WLAN+BT BCM94352HMB](https://www.ebay.fr/itm/Azurewave-AW-CE123H-Broadcom-BCM94352-802-11ac-WiFi-card-Bluetooth-4-0-for-MAC-/272248789669) ebay
- Wifi mini PCI Wifi mini PCI [AzureWave WLAN+BT BCM94352HMB](https://www.aliexpress.com/item/4000385703330.html) aliexpress

- (Broadcom BCM943224HMS working for Monterey 12.1+) with boot args `brcmfx-delay=15000` Use the boot args only with Monterey 12 ☞ Wifi mini PCI Wifi mini PCI [Broadcom BCM943224HMS](https://www.ebay.com/itm/253022938877?hash=item3ae957a6fd:g:8JYAAMXQVT9TFoF4) 
- This card suppose to be work again in macOS Lower thene Monterey 12 but not testing by me for long time a go.

**Reply by MANALICANTE** [reply]
I already read this post, but it must be from HP? Are these models accepted by the white list?

thank you

**Comment by chris1111** [comment]
Look this at the seller card,  is HP Broadcom BCM943224HMS

[Broadcom BCM943224HMS](https://www.ebay.com/itm/253022938877?hash=item3ae957a6fd:g:8JYAAMXQVT9TFoF4) 

![Screenshot 2023-02-19 at 8 01 41 AM](https://user-images.githubusercontent.com/6248794/219949706-babc8e09-37ba-45cb-9043-105e0ef5c181.png)

**Reply by MANALICANTE** [reply]
OK!!!!! thank you Chris!!!
